src/auto-evo/RunResults.cs

```diff
@@ -584,9 +584,10 @@ public void PrintSummary(PatchMap? previousPopulations = null)
         ///   Must be final effects with <see cref="ExternalEffect.Coefficient"/> set to 1 created by
         ///   <see cref="AutoEvoRun.CalculateFinalExternalEffectSizes"/>
         /// </param>
+        /// <param name="ignoreFogOfWar">If ture ignore fog of war</param>
         /// <returns>The generated summary text</returns>
         public LocalizedStringBuilder MakeSummary(PatchMap? previousPopulations = null,
-            bool playerReadable = false, List<ExternalEffect>? effects = null)
+            bool playerReadable = false, List<ExternalEffect>? effects = null, bool ignoreFogOfWar = false)
```

Comment:
I think maybe the player readable being false should also imply ignoreFogOfWar as that is meant for a technical data dump, and obfuscating any data in that doesn't make any sense to me...